ui/narrative/methods/annotate_contigs/display.yaml

```diff
@@ -19,7 +19,7 @@ suggestions:
             []
     methods:
         related:
-            []
+            [annotate_contigset, annotate_domains_in_a_genome, annotate_plant_transcripts, reannotate_microbial_genome]
```

Comment:
You can check module name in kbase.yml file: https://github.com/kbaseapps/RAST_SDK/blob/master/kbase.yml#L2